katharsis-rs/src/main/java/io/katharsis/rs/KatharsisFilter.java

```diff
@@ -154,12 +154,17 @@ private void dispatchRequest(ContainerRequestContext requestContext) throws Exce
             	// no repositories invoked, we do nothing and forward the call to JAX-RS
             	passToMethodMatcher = true;
             }
+       
         } catch (KatharsisMappableException e) {
+        	e.printStackTrace();
             // log error in KatharsisMappableException mapper.
             katharsisResponse = new KatharsisExceptionMapper().toErrorResponse(e);
         } catch (KatharsisMatchingException e) {
+        	e.printStackTrace();
         	LOGGER.warn("failed to process request", e);
             passToMethodMatcher = true;
+        } catch (Throwable e) {
```

Comment:
sonar setup is not workign anymore? usually it taskes care of this kind of things.